v03_pipeline/lib/reference_data/config.py

```diff
@@ -52,10 +54,21 @@ def clinvar_custom_select(ht):
     # NB: the `enum_select` does not support mapping a list of tuples
     # so there's a hidden enum-mapping inside this clinvar function.
     selects['conflictingPathogenicities'] = parsed_and_mapped_clnsigconf(ht)
-    selects['goldStars'] = CLINVAR_GOLD_STARS_LOOKUP.get(hl.delimit(ht.info.CLNREVSTAT))
+    selects['goldStars'] = CLINVAR_GOLD_STARS_LOOKUP.get(hl.delimit(ht.CLNREVSTAT))
     return selects
 
 
+def join_clinvar_to_submission_summary(ht, selects: dict):
+    submission_ht = download_and_import_clinvar_submission_summary()
+    ht = ht.info.annotate(
+        submitters=submission_ht[ht.rsid].Submitters,
+        conditions=submission_ht[ht.rsid].Conditions,
+    )
+    selects['submitters'] = ht.submitters
```

Comment:
it's maybe _slightly_ easier to understand this if the `selects` stay in `clinvar_custom_select`!

Reply:
This was another choice that makes mocking easier - our tests were already running the `clinvar_custom_select` code so it was easiest to put the submitter/condition stuff in a new function, and in tests mock the new function to return `ht.info`.

Reply:
oh wait I see what you mean here. I can annotate without passing and adding to `selects`